src/edu/washington/escience/myria/operator/agg/CountAllAggregator.java

```diff
@@ -53,4 +56,9 @@ public Object getInitialState() {
     /** The number of tuples seen so far. */
     private long count = 0;
   }
+
+  @Override
+  public void add(final List<TupleBatch> from, final Object state) throws DbException {
+    throw new DbException(" method not implemented");
```

Comment:
we could implement it, since counting will never require  a python expression, the issue of inefficiency due to writing state multiple times to an external process is avoided.